Gemfile

```diff
@@ -69,7 +69,7 @@ group :test do
   gem 'launchy'
   gem 'rack-test', require: 'rack/test'
   gem 'mocha', require: false
-  gem 'bourne', require: false
+  gem 'spy'
```

Comment:
should we keep `require: false` here?

Reply:
I think since it's only in the bundler `test` group, and is always going to be used when the tests run, it's probably fine to have it required by bundler.